### PR TITLE
Tiny improvements

### PR DESCRIPTION
**Pull request by eloycoto**
Two commits in this PR, not sure if it's the way that you used to work, but detailed information on each commit message:

1) Small change on Makefile to make it easier to run on fedora.
2) Small logging changes to add context to error messages.

Thanks!

### CODE REVIEW

File: cmd/device-worker/main.go [hunk 1]
```diff
@@ -87,7 +87,7 @@ func main() {
 
 	wl, err := workload2.NewWorkloadManager(dataDir)
 	if err != nil {
-		log.Fatal(err)
+		log.Fatal("Cannot started Workload Manager, err: ", err)
```
Comment (jakub-dzon) [inline cmd/device-worker/main.go:90]: "Cannot start ..."

File: cmd/device-worker/main.go [hunk 2]
```diff
@@ -74,7 +74,7 @@ func main() {
 	// Listen on the provided socket address.
 	l, err := net.Listen("unix", r.GetAddress())
 	if err != nil {
-		log.Fatal(err)
+		log.Fatal("Cannot start listening on %s err: ", r.GetAddress(), err)
```
Comment (jakub-dzon) [inline cmd/device-worker/main.go:77]: I think that should be `Fatalf`.

**Review by jakub-dzon — CHANGES_REQUESTED** [review]
Thank you for the first contribution!

File: cmd/device-worker/main.go (hunk 2 above)
Comment (masayag) [reply cmd/device-worker/main.go:77]: I think there will be a mismatch between the number of arguments in the format string to the number of provided arguments.
Perhaps change to:
```
log.Fatalf("Cannot start listening on %s err: %v", r.GetAddress(), err)
```

File: cmd/device-worker/main.go (hunk 1 above)
Comment (eloycoto) [reply cmd/device-worker/main.go]: Fixed sorry!

File: cmd/device-worker/main.go (hunk 2 above)
Comment (eloycoto) [reply cmd/device-worker/main.go]: Fixed sorry!

File: cmd/device-worker/main.go [hunk 3]
```diff
@@ -74,7 +74,7 @@ func main() {
 	// Listen on the provided socket address.
 	l, err := net.Listen("unix", r.GetAddress())
 	if err != nil {
-		log.Fatal(err)
+		log.Fatalf("Cannot start listening on %s err: ", r.GetAddress(), err)
```
Comment (masayag) [inline cmd/device-worker/main.go:77]: I think there will be a failure in printing the message due to a mismatch between the number of expected argument in the format message (1) to the actual number or provided arguments (2)

**Review by masayag — APPROVED** [review]

**Pull request merged**